apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/pipeline/impl/global_nn_recognizer_cvfh.hpp

```diff
@@ -22,8 +22,7 @@ template<template<class > class Distance, typename PointInT, typename FeatureT>
       typedef std::pair<std::string, int> mv_pair;
       mv_pair pair_model_view = std::make_pair (model.id_, view_id);
 
-      std::map<mv_pair, Eigen::Matrix4f, std::less<mv_pair>, Eigen::aligned_allocator<std::pair<mv_pair, Eigen::Matrix4f> > >::iterator it =
-          poses_cache_.find (pair_model_view);
+      auto it = poses_cache_.find (pair_model_view);
```

Comment:
Unfortunately we can only adopt this once we move to C++14. Right now this is not supported.

Reply:
`auto` is supported since C++11, not C++14.
That's why I used it. Anyway, if you want me to revert this to its previous type declaration I can do it.

Reply:
PCL is currently on C++03, not C++11. We're skipping C++11 entirely and going straight for C++14.

Reply:
> PCL is currently on C++03, not C++11. We're skipping C++11 entirely and going straight for C++14.

Ok, now it's also clear to me why the topic about C++14. There's a skip of C++11 that I missed altogether 😄 

Roger. Reverting the iterator right now.